### PR TITLE
Included Application Guard

### PR DESCRIPTION
**Pull request by ekarafezov**
Application Guard is an upgraded experience (using SafeDocs as well) from Protected View when tenant has enabled the feature and set the policy.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/safe-docs.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ ms.prod: m365-security
 - [Microsoft Defender for Office 365 plan 2](defender-for-office-365.md)
 - [Microsoft 365 Defender](../defender/microsoft-365-defender.md)
 
-Safe Documents is a feature in Microsoft 365 E5 or Microsoft 365 E5 Security that uses [Microsoft Defender for Endpoint](/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-advanced-threat-protection) to scan documents and files that are opened in [Protected View](https://support.microsoft.com/office/d6f09ac7-e6b9-4495-8e43-2bbcdbcb6653).
+Safe Documents is a feature in Microsoft 365 E5 or Microsoft 365 E5 Security that uses [Microsoft Defender for Endpoint](/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-advanced-threat-protection) to scan documents and files that are opened in [Protected View](https://support.microsoft.com/office/d6f09ac7-e6b9-4495-8e43-2bbcdbcb6653) or [Application Guard for Office](https://support.microsoft.com/en-us/topic/application-guard-for-office-9e0fb9c2-ffad-43bf-8ba3-78f785fdba46).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/safe-docs.md]:
```suggestion
Safe Documents is a feature in Microsoft 365 E5 or Microsoft 365 E5 Security that uses [Microsoft Defender for Endpoint](/windows/security/threat-protection/microsoft-defender-atp/microsoft-defender-advanced-threat-protection) to scan documents and files that are opened in [Protected View](https://support.microsoft.com/office/d6f09ac7-e6b9-4495-8e43-2bbcdbcb6653) or [Application Guard for Office](https://support.microsoft.com/topic/application-guard-for-office-9e0fb9c2-ffad-43bf-8ba3-78f785fdba46).
```

File: microsoft-365/security/office-365-security/safe-docs.md [hunk 2]
```diff
@@ -90,6 +90,11 @@ Set-AtpPolicyForO365 -EnableSafeDocs $true -AllowSafeDocsOpen $false
 
 For detailed syntax and parameter information, see [Set-AtpPolicyForO365](/powershell/module/exchange/set-atppolicyforo365).
 
+### Onboard to the Microsoft Defender for Endpoint Service to enable auditing capabilities
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/safe-docs.md:93]:
```suggestion
### Onboarding to the Microsoft Defender for Endpoint service to enable auditing capabilities
```

File: microsoft-365/security/office-365-security/safe-docs.md [hunk 3]
```diff
@@ -90,6 +90,11 @@ Set-AtpPolicyForO365 -EnableSafeDocs $true -AllowSafeDocsOpen $false
...
+
+To deploy Microsoft Defender for Endpoint you must go through the various phases of deployment. After onboarding you can configure auditing capabilities in the Security & Compliance Center.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/safe-docs.md]:
```suggestion
To deploy Microsoft Defender for Endpoint you must go through the various phases of deployment. After onboarding, you can configure auditing capabilities in the Security & Compliance Center. 
```

File: microsoft-365/security/office-365-security/safe-docs.md [hunk 4]
```diff
@@ -90,6 +90,11 @@ Set-AtpPolicyForO365 -EnableSafeDocs $true -AllowSafeDocsOpen $false
 
 For detailed syntax and parameter information, see [Set-AtpPolicyForO365](/powershell/module/exchange/set-atppolicyforo365).
 
+### Onboard to the Microsoft Defender for Endpoint Service to enable auditing capabilities
+
+To deploy Microsoft Defender for Endpoint you must go through the various phases of deployment. After onboarding you can configure auditing capabilities in the Security & Compliance Center. 
+To learn more see [Onboard to the Microsoft Defender for Endpoint service] (/security/defender-endpoint/onboarding). If you need additional help please refer to the [Troubleshoot Microsoft Defender for Endpoint onboarding issues] (/security/defender-endpoint/troubleshoot-onboarding).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/safe-docs.md]:
```suggestion
To learn more, see [Onboard to the Microsoft Defender for Endpoint service] (/security/defender-endpoint/onboarding). If you need additional help, please refer to [Troubleshoot Microsoft Defender for Endpoint onboarding issues](/security/defender-endpoint/troubleshoot-onboarding).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@ekarafezov Please implement the feedback. Thanks

**Pull request merged**